templates/quickstart-postgres-for-atlassian-services.yaml

```diff
@@ -15,7 +15,7 @@
 # DBAllocatedStorageEncrypted
 # DBStorageType
 AWSTemplateFormatVersion: 2010-09-09
-Description: RDS Postgres Database for use in Atlassian Standard Infrastructure
+Description: RDS Postgres Database for use in Atlassian Standard Infrastructure (qs-1aj6s44e4)
```

Comment:
What's `(qs-1aj6s44e4)` in the description for?

Reply:
They were added by Tony in upstream. I have just synced Postgres template and didn't sync Aurora. I will fix it.

Updated version:
https://github.com/aws-quickstart/quickstart-atlassian-services/blob/develop/templates/quickstart-postgres-for-atlassian-services.yaml

Documentation:
https://aws-quickstart.github.io/qs-ids.html
(they are generated by AWS)

Reply:
I have synced `develop` with upstream aaand they are gone (from the PR).